.gitignore

```
@@ -99,3 +99,4 @@ doc/*.build
 src/x_build_libs.ocp
 # dev setup
 /.ocamlinit
+release/out
```

```suggestion
/release/out
```
otherwise it matches all directories `out` with a parent directory `release`